src/app/components/LongDataDisplay/index.tsx

```diff
@@ -56,23 +55,17 @@ export const LongDataDisplay: FC<{ data: string; fontWeight?: number; collapsedL
 
   return (
     <div>
-      <Typography
-        variant="mono"
+      <span
         ref={textRef}
-        sx={{
-          fontWeight,
-          maxHeight: isExpanded ? 'none' : collapsedContainerMaxHeight,
-          overflow: 'hidden',
+        className="font-medium overflow-hidden overflow-wrap-anywhere whitespace-pre-wrap [display:-webkit-box][WebkitBoxOrient:vertical]"
```

Comment:
This doesn't seem to be working
https://explorer.dev.oasis.io/mainnet/sapphire/tx/0xedfa6dc07686568299574ccad7eaa2b054b9f0062ed61b824f79860dfc67cf2b

Before | After
<img width="1002" height="503" alt="image" src="https://github.com/user-attachments/assets/1c82d310-bfcb-4a19-b067-e6e6e9b992a5" />